hr_timesheet_sheet_validators/model/hr_timesheet_sheet.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2010 Tiny SPRL (<http://tiny.be>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import fields, osv
+from openerp.tools.translate import _
+from openerp import netsvc
+
+
+class hr_timesheet_sheet(osv.osv):
+    _inherit = "hr_timesheet_sheet.sheet"
+
+    _columns = {
+        'validator_user_ids': fields.many2many('res.users', string='Validators',
+                                               required=False),
+    }
+
+    def _default_department(self, cr, uid, context=None):
+        emp_obj = self.pool.get('hr.employee')
+        emp_ids = emp_obj.search(cr, uid, [('user_id', '=', uid)],
+                                 context=context)
+        emps = emp_obj.browse(cr, uid, emp_ids, context=context)
+
+        for emp in emps:
+            return emp.department_id and emp.department_id.id or False
+        return False
+
+    _defaults = {
+        'department_id': _default_department,
+    }
+
+    def _get_validator_user_ids(self, cr, uid, ids, context=None):
+        res = {}
+        users = {}
+        for timesheet in self.browse(cr, uid, ids):
+            res[timesheet.id] = []
+            users[timesheet.id] = []
+            if timesheet.employee_id \
```

Comment:
You should consider to use brackets instead of ```\```
I mean something like this:

```
if  (
    timesheet.employee_id
    and timesheet.employee_id.parent_id
    and timesheet.employee_id.parent_id.user_id
):
```